Extension/src/LanguageServer/extension.ts

```diff
@@ -128,7 +128,7 @@ async function getBuildTasks(): Promise<vscode.Task[]> {
         activeFileLanguage = 'cpp';
     } else if (activeFileExt === '.c') {
         activeFileLanguage = 'c';
-    } else if (activeFileExt !== '.C') { // Ambiguous file extension
+    } else if (activeFileExt !== '.C') { // Ambiguous file extension.
```

Comment:
ambiguous means show tasks for both c and c++

Reply:
I updated the comment to reflect that's the case